workflow/rules/common.smk

```diff
@@ -29,7 +29,7 @@ def cd(path, logger):
 
 ##### load config and sample sheets #####
 configfile: "config/config.yaml"
-
+configfile: "config/envmodules.yaml"
```

Comment:
Since not all users will be using environment modules, we should wrap this in an if statement:
```
if os.path.exists("config/envmodules.yaml"):
    configfile: "config/envmodules.yaml"
```

Reply:
Notice that I have pushed an empty `config/envmodules.yaml` file in this PR since I think it would be useful for the user to see examples of how it should be setup in case they moved to HPCs. In that case the file would always be there and validated, let me know if you don't believe this is a good idea

Reply:
(never mind that I had forgotten to actually put `envmodules.yaml` inside `.test/config/`)

Reply:
+1 on adding empty envmodules.yaml files. One can actually set default values in the schema, but better explicit than implicit. I'm still inclined towards adding the if statement though since 1) I'm not sure it is evident that an end user knows that you *must* add '{}' to an empty yaml file (even though the example file would demonstrate that fact) and 2) environment modules are really only used in certain circumstances. We can do this at a later stage though as our current use case will entail the use of envmodules.